index.html

```diff
@@ -9,118 +9,116 @@
       name="description"
       content="Keri West's website is a digital portfolio to display his coding work and a little about himself."
     />
+
     <title>Keri West's Website</title>
-    <link rel="stylesheet" type="text/css" href="styles.css" />
+
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
+    <link rel="stylesheet
+    href="https://fonts.google.com/specimen/Belanosima?query=Belanosima"> <link
+    rel="stylesheet
+    href="https://fonts.google.com/specimen/Roboto+Mono?query=roboto+m">
```

Comment:
```suggestion
    <link rel="stylesheet"
    href="https://fonts.google.com/specimen/Belanosima?query=Belanosima">
    <link rel="stylesheet"
    href="https://fonts.google.com/specimen/Roboto+Mono?query=roboto+m">
```

Missing closing quote, so the fonts aren't loading: Instead of seeing `rel="stylesheet" ...`, it sees `rel="stylesheet href" ...`. (Unconventional line breaks make this harder to see.)

Also, the URLs you're using are for the human readable HTML pages: The don't actually deliver the CSS for the fonts. See https://developers.google.com/fonts/docs/css2